docs/install/grafana-setup.rst

```diff
@@ -1,26 +1,26 @@
 .. meta::
-   :description: Omniperf Grafana server installation and deployment
-   :keywords: Omniperf, ROCm, profiler, tool, Instinct, accelerator, AMD,
+   :description: ROCm Compute Profiler Grafana server installation and deployment
+   :keywords: ROCm Compute Profiler, ROCm, profiler, tool, Instinct, accelerator, AMD,
               install, deploy, Grafana, server, configuration, GUI
 
-****************************************
-Setting up a Grafana server for Omniperf
-****************************************
+***************************************************
+Setting up Grafana server for ROCm Compute Profiler
```

Comment:
Many of the images here still show Omniperf.

Reply:
I think it's ok to leave those for now